statcode/statcode.py

```diff
@@ -109,6 +100,10 @@ def render(self, size, focus=False):
 
         return canv
 
+    def mouse_event(self, size, event, button, col, row, focus):
```

Comment:
Awesome! I've been trying to figure this out on a variety of urwid projects now, thanks.